setup.py

```diff
@@ -35,7 +35,7 @@
 except ImportError:
     pass
 
-requires = ['satpy <= 0.37, > 0.41.1', 'pyorbital', 'trollsift', 'pyspectral', 'h5netcdf'],
+requires = ['satpy!=0.38.*,!=0.39.*,!=0.40.*,!=0.41.*', 'pyorbital', 'trollsift', 'pyspectral', 'h5netcdf'],
```

Comment:
Should it note be !=0.41.0 as 0.41.1 is fine?

Reply:
The original `requires` line says `> 0.41.1` so `0.41.1` was already blocked, so went with that.

Reply:
Ok!